scripts/increment-version.js

```diff
@@ -7,30 +7,19 @@ const fs = require('fs');
 const path = require('path');
 const packageJson = require('../package.json');
 
-// Setup auth
-fs.writeFileSync(`${process.env['HOME']}/.npmrc`, `//registry.npmjs.org/:_authToken=${process.env['NPM_AUTH_TOKEN']}`);
-
 // Determine if this is a stable or beta release
 const publishedVersions = getPublishedVersions();
-const isStableRelease = publishedVersions.indexOf(packageJson.version) === -1;
+const isStableRelease = !publishedVersions.includes(packageJson.version);
 
 // Get the next version
-let nextVersion = isStableRelease ? packageJson.version : getNextBetaVersion();
-console.log(`Publishing version: ${nextVersion}`);
+const nextVersion = isStableRelease ? packageJson.version : getNextBetaVersion();
+console.log(`Setting version to ${nextVersion}`);
 
 // Set the version in package.json
 const packageJsonFile = path.resolve(__dirname, '..', 'package.json');
 packageJson.version = nextVersion;
 fs.writeFileSync(packageJsonFile, JSON.stringify(packageJson, null, 2));
 
-// Publish
-const args = ['publish'];
-if (!isStableRelease) {
-  args.push('--tag', 'beta');
```

Comment:
I believe the pipeline will have to support this tag functionality as well. Leaving the PR unmerged.

Reply:
Yes, we automatically create a GH release and tag.

Reply:
I have added tag parameters to the YAML that should tell ESRP to publish with the correct tag.